.dockerignore

```diff
@@ -12,7 +12,6 @@ examples/
 
 integration/
 integration-scripts/
-integration-tests/
```

Comment:
This is a must since we have a local `replace` I need it to be included if I build `Dockerfile` from the root dir context.

Reply:
Which Dockerfile are you using? You can have a dockerignore setup for a specific Dockerfile so we don't have to mess with core's dockerignore.

Reply:
Good catch, added default `.gitignore` for WASP image.